src/python/grpcio/grpc/framework/common/BUILD.bazel

```diff
@@ -25,17 +25,11 @@ py_library(
 py_library(
     name = "cardinality",
     srcs = ["cardinality.py"],
-    deps = select({
-        "//conditions:default": ["@enum34//:enum34"],
-        "//:python3": [],
-    }),
+    deps = [],
```

Comment:
Nit: You can just remove `deps` now that it's empty. Here and throughout.

Reply:
Git it, will fix in next revision.